### PR TITLE
add styles for safety screen, alert, water parameters

### PR DESCRIPTION
**Pull request by devfunmilayo**

### CODE REVIEW

File: components/safety-screen/safety-screen.component.tsx [hunk 1]
```diff
@@ -1,7 +1,31 @@
-import { SafeScreenContainer } from "../../styles/safety-screen.styles";
+import { ImageContainer, SafeScreenContainer, TextContainer } from "../../styles/safety-screen.styles";
```
Comment (ini-s) [inline components/safety-screen/safety-screen.component.tsx:1]: Why did you import from the same file twice? Line 1 and Line 2 are from the same file so join them together

File: components/water-parameters/water-parameters.component.tsx [hunk 2]
```diff
@@ -1,7 +1,116 @@
-import { WaterParametersContainer } from "../../styles/water-parameters.styles";
+import React from "react";
+import {
+  WaterParametersContainer,
+  CombinedContainer,
+} from "../../styles/water-parameters.styles";
+import Image from "next/image";
```
Comment (ini-s) [inline components/water-parameters/water-parameters.component.tsx:6]: Place this Image import after line 1 and then leave a space after.

File: styles/alert.styles.tsx [hunk 3]
```diff
@@ -1,4 +1,117 @@
...
+  max-width: 45.6875rem;
+  padding: 1rem;
+  //padding-top: -2rem;
+  border: 1px solid rgba(255, 255, 255, 1);
+  box-shadow: 0 0 0.625rem 0, 0, 0;
+  text-align: left;
+  background-color: #fae1007d;
+  padding-bottom: 1rem;
+  box-sizing: border-box;
+  //margin-top: 2.5rem;
```
Comment (ini-s) [inline styles/alert.styles.tsx:15]: Please remove all commented out styles

File: styles/alert.styles.tsx [hunk 4]
```diff
@@ -1,4 +1,117 @@
...
+    // margin-top: 62rem;
+    margin-left: 0.1rem;
+    height: auto;
+  }
+
+  @media (max-width: 480px) {
+    padding: 1rem;
+    margin-top: 1rem;
+  }
+  @media (max-width: 1400px) {
```
Comment (ini-s) [inline styles/alert.styles.tsx:29]: change all media queries from px to em. Em has the same value as rem so let your vscode extension do the conversion and then you remove the r from rem. Do this for all media queries on all the components you're working on.

File: styles/safety-screen.styles.ts [hunk 5]
```diff
@@ -1,3 +1,120 @@
 import styled from "styled-components";
 
-export const SafeScreenContainer = styled.section``;
+export const SafeScreenContainer = styled.div`
```
Comment (ini-s) [inline styles/safety-screen.styles.ts:3]: I name SafeScreenContainer as a section not a div, why did you change it to a div?

Comment (ini-s) [inline styles/safety-screen.styles.ts:3]:
```suggestion
export const SafeScreenContainer = styled.section`
```

File: styles/water-parameters.styles.ts [hunk 6]
```diff
@@ -1,3 +1,142 @@
 import styled from "styled-components";
 
-export const WaterParametersContainer = styled.section``;
```
Comment (ini-s) [inline styles/water-parameters.styles.ts:3]: the style for combined container, is it to flex the alert and water parameters component? If so it should be in the water-quality-data.styles.ts
So create a file under the styles folder and name it water-quality-data.styles.ts
Don't forget to import styled components into that file first before you start styling

**Review by ini-s — APPROVED** [review]

File: components/safety-screen/safety-screen.component.tsx (hunk 1 above)
Comment (devfunmilayo) [reply components/safety-screen/safety-screen.component.tsx:1]: Alright, I've done that

Comment (ini-s) [reply components/safety-screen/safety-screen.component.tsx:1]: Okay so when you've done something click on "Resolve conversation" below
